libs/get-locale/index.js

```diff
@@ -0,0 +1,53 @@
+const { parse } = require("cookie");
+const acceptLanguageParser = require("accept-language-parser");
+
+const {
+  DEFAULT_LOCALE,
+  VALID_LOCALES,
+  PREFERRED_LOCALE_COOKIE_NAME,
+} = require("../constants");
+const VALID_LOCALES_LIST = [...VALID_LOCALES.values()];
+
+// From https://github.com/aws-samples/cloudfront-authorization-at-edge/blob/01c1bc843d478977005bde86f5834ce76c479eec/src/lambda-edge/shared/shared.ts#L216
```

Comment:
Based on this copy-n-paste I feel pretty confident that this is how cookies are parsed in Lambda@Edge. But it's honestly just speculation to be honest. The true test is to actually test it once the Lambda version upgrades and we update the version in the CloudFront distribution.